src/Jira.js

```diff
@@ -22,7 +22,7 @@ export default class Jira {
     this.ticketPromises = {};
 
     const { host, username, password} = config.jira.api;
-    let { email, token } = config.jira.api;
+    let { email, token, originalOptions } = config.jira.api;
```

Comment:
Let's just call it `options` for carity.

Reply:
@jgillick please check if that's fine now